api/controllers/v2/slack.ts

```diff
@@ -99,6 +100,13 @@ export const DisplayRepoOptions = async (event: APIGatewayEvent): Promise<APIGat
 async function deployRepo(deployable: Array<any>, logger: ILogger, jobRepository: JobRepository, jobQueueUrl) {
   try {
     await jobRepository.insertBulkJobs(deployable, jobQueueUrl);
+
+    // we need to also notify the jobUpdatesQueue as well so that slack users get notified
+    await Promise.all(
+      deployable.map(async ({ jobId }) => {
+        await jobRepository.notify(jobId, c.get('jobUpdatesQueueUrl'), JobStatus.inQueue, 0);
```

Comment:
i see we have some different places for `jobRepository.notify` vs sqs.sendMessage directly. will this be consolidated. ie. can queue collection and sqs events be simultaneous or does it have to be split up ie. [job controller TriggerLocalBuild](https://github.com/mongodb/docs-worker-pool/pull/831/files#diff-c45958abd9415b796244840efc442d191113b562eed4712968ef25fd51429d72R33)

Reply:
I am not entirely sure why in certain cases the logic is separated out, but I am not going to consolidate them at this point. Right now, I am trying to as closely mimic the original logic so that I don't introduce unintended bugs or behaviors. Hopefully, this can be addressed at another point.